src/main/java/taxi/controller/IndexController.java

```diff
@@ -5,11 +5,21 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.DriverService;
 
 public class IndexController extends HttpServlet {
+    private static final Injector injector = Injector.getInstance("taxi");
+    private final DriverService driverService = (DriverService) injector
+            .getInstance(DriverService.class);
+
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
+        Long currentDriverId = (Long) req.getSession().getAttribute("driver_id");
+        Driver driver = driverService.get(currentDriverId);
+        req.setAttribute("currentDriverName", driver.getName());
```

Comment:
I think it's better to store driver name in session, not to make db request every time user visit home page